tests/testthat/test-gefs.R

```diff
@@ -1,61 +1,80 @@
 context("gefs")
 
 #set a location
-lat = 46.28125
-lon = -116.2188
+lons <- c(-1.1, 0, 0.8, 2)
+lats <- c(50.1, 51, 51.9, 53, 54)
 
 #variable
-var = "Temperature_height_above_ground_ens"
+temp = "Temperature_height_above_ground_ens"
+
+# Get raw and processed data
+d_raw <- gefs(var = temp,
```

Comment:
I'd like any code in tests that make HTTP requests to be skipped on cran because any http request failure could cause cran maintainers to throw a tantrum

Reply:
you can do variable assignment up here but not HTTP requests

Reply:
Right, I knew there was a reason I had it differently before. Ultimately I should move those to mock api calls but for now I've moved them inside of the test and put all tests related to the returned data object inside of one `test_that()` call.

Reply:
thanks